.github/workflows/ci.yaml

```diff
@@ -34,6 +34,10 @@ jobs:
             spog/ui/target/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
+      - name: Add build dependencies
+        run: |
+          sudo apt install clang llvm pkg-config nettle-dev
```

Comment:
I switched the build to use openssl in the other PR. I think it makes sense to stick to openssl, unless there's a good reason to fiddle with nettle.

Reply:
Agreed, I just followed the recommendation from the build failure, didn't realise was an openssl alternative.